src/transform/EditTransform.js

```diff
@@ -64,11 +71,63 @@ const newEditSectionHeader = (document, index, level, titleHTML, showEditPencil
     element.appendChild(button)
   }
 
+  if (anchor && anchor.length > 0) {
+    // TODO: consider renaming this 'id' to 'anchor' for clarity - would need to update native
+    // code as well - used when TOC sections made to jump to sections.
+    element.id = anchor
+  }
+
   return element
 }
 
+/**
+ * Lead section header is a special case as it needs to show article title and description too,
+ * and in addition to the lead edit pencil, the description can also be editable.
+ * As a client, you may wish to set the ID attribute.
+ * @param {!Document} document
+ * @param {?string} articleDisplayTitle Article display title.
+ * @param {?string} titleDescription Article title description.
+ * @param {?string} addTitleDescriptionString Localized string e.g. 'Add title description'.
+ * @param {?boolean} isTitleDescriptionEditable Whether title description is editable.
+ * @param {?boolean} showEditPencil Whether to show the "edit" pencil (default is true).
+ * @param {?string} anchor Section anchor used for jumping between sections.
+ * @return {!HTMLElement}
+ */
+const newEditLeadSectionHeader = (document, articleDisplayTitle, titleDescription,
+  addTitleDescriptionString, isTitleDescriptionEditable, showEditPencil = true, anchor) => {
```

Comment:
When would this be false?

Reply:
On iOS right now the lead section edit pencil doesn't appear directly to the right of the article title. It's further down. I left the parameter in for now so iOS can immediately start using this transform even before we switch over to PCS.

Reply:
Ah, I see. Thanks!